core/model/modx/modx.class.php

```diff
@@ -524,6 +524,18 @@ public function initialize($contextKey= 'web', $options = null) {
             $this->getCacheManager();
             $this->getConfig();
             $this->_initContext($contextKey, false, $options);
+            $logTarget = $this->getLogTarget();
+            if ($logTarget === 'FILE') {
+                $options = array();
+                $filename = $this->getOption('error_log_filename', $options, '');
+                if (!empty($filename)) $options['filename'] = $filename;
```

Comment:
Not a strictly required but it's better to follow PSR rules and use `if` with {} to avoid unexpected behavior in the future somehow.